datasets/sem_eval_2010_task_8/sem_eval_2010_task_8.py

```diff
@@ -0,0 +1,139 @@
+# coding=utf-8
```

Comment:
Could you make sure this PR's changes don't include changes to other datasets than LER ?

Reply:
Sure. Sorry, my bad. Made the branch out of the previous dataset I added instead of master.
Thanks for pointing it out